python/neuroglancer/chunks.py

```diff
@@ -17,13 +17,40 @@
 import numpy as np
 from PIL import Image
 
+def encode_jpeg(subvol):
+    shape = subvol.shape
+    reshaped = subvol.reshape(shape[0] * shape[1], shape[2])
+    img = Image.fromarray(reshaped)
+    f = io.BytesIO()
+    img.save(f, "JPEG")
+    return f.getvalue()
+
+def decode_jpeg(bytestring, shape=(64,64,64), dtype=np.uint8):
+
+    if len(bytestring) <= 3404: # black jpeg of shape (64,64,64) in mode L
```

Comment:
I was running into problems where the name of the file didn't match the size of the data. I used heuristics to try to deal with these problems. This was a heuristic for detecting totally black tiles. If a tile is totally black, I can reshape it at will without worrying about losing data.